### ISSUE TITLE
Is it possible to use Compass Sprites when not using Compass?

### ISSUE THREAD

**Issue by jmuheim** [hagenburger/lemonade#35]
Hey there

Sad to see your Lemonade gone. We don't move SASS/Compass as we are using Twitter Bootstrap, but I wonder whether it's possible to use the Sprites functionalities of Compass on its own?

Thank you.

**Comment by hagenburger** [comment]
No. Both Sass and Compass are a requirement for creating the sprites. But you are free to use the spriting parts of Compass only. There’s no need to use the CSS3 mixins.
